src/components/files-upload-overview.yml

```diff
@@ -7,4 +7,18 @@ uploadLimitExceeded:
 insufficientPrivateStorge:
   en: Not enough storage space available for file
   de: Nicht genügend Speicherplatz für die Datei verfügbar
-
+startUpload:
+  en: Start upload
+  de: Upload starten
+preprocessed:
+  en: edited
+  de: bearbeitet
+stage_uploadNotStarted:
+  en: The following file will be uploaded. You can edit images before upload by clicking on the pen icon.
```

Comment:
The following file**s** [..]

Reply:
I changed it using the fancy pluralization feature in our translations :-)